src/components/clusterexplorer/resourcekinds/resourcekind.pod.ts

```diff
@@ -13,14 +13,21 @@ export const podUICustomiser = {
     customiseTreeItem(resource: ResourceNode, treeItem: vscode.TreeItem): void {
         const podInfo = resource.extraInfo!.podInfo;  // TODO: unbang
         if (podInfo && podInfo.status) {
-            treeItem.iconPath = getIconForPodStatus(podInfo.status.toLowerCase());
+            treeItem.iconPath = getIconForPodStatus(podInfo.status.toLowerCase(), podInfo.ready);
         }
     }
 };
 
-function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
-        return assetUri("images/runningPod.svg");
+function getIconForPodStatus(status: string, ready: string): vscode.Uri {
+    if (status === "running") {
+        const readyFields = ready.split("/", 2);
```

Comment:
💡 Thank you so much for adding to this review ❤️🙏 Idea / Just some thought to share: Might be if we add comment with regards to what is going on here: aka - return sample `ready: string` expected value and why only limit of 2, also, will this separator be different base on O/S. Thanks.

Reply:
Thank you @Tatsinnit for the review.  I will add the needed comment.  I am not sure I follow the question about the limit of two.  I am expecting a string that includes the number of ready and total containers separated by a / character (which I see on macOS, Windows, and Linux).  I am guessing you know something that I have missed about the Ready column values or perhaps the string split method.  Can you please elaborate on the limit of 2 question?

Reply:
Thank you so much @carljmosca, sounds good yeah, further clarification why I shared the observation is below.

- Regarding path - yeah all good, thought I will mention - hence I was sharing/wondering if use of `os.sep` or `path.sep` which could take care of any discrepancy in cases where the path do not have `/`. https://stackoverflow.com/questions/125813/how-to-determine-the-os-path-separator-in-javascript 
- Re: **Limit** - what I was meaning to share the observation of use `split([separator][, limit]);` so the limit of `2` will return first `2` and I think new images are in location `images/newimages.svg` so should be good -  : https://jsfiddle.net/75k1pt8z/  

Thanks heaps and I thought I will share this just as fyi in case. 🙏❤️